_members/cameron-mattson.md

```diff
@@ -0,0 +1,26 @@
+---
+name: Cameron Mattson
+image: images/team/cameron_mattson.jpg
+description: Data Scientist
+role: programmer
+aliases:
+  - Cameron Mattson
+  - Cameron W. Mattson
+  - C. Mattson
+  - cmattson
+  - mcameron
+links:
+  email: cameron.mattson@cuanschutz.edu
+  github: https://github.com/MattsonCam
+---
+
+Cameron graduated with his B.S. in Bioengineering at the University of Colorado, where he implemented and developed physiological models.<br>
```

Comment:
please remove the line breaks (`<br>`)

It makes the deploy look a bit wonky.

I'll merge after! thanks!

Reply:
Sounds good, changed!